### PR TITLE
Fix: Add Text Generation model for Keras LSTM Demo (#115)

### PR DESCRIPTION
**Pull request by Ram81**
@dexter1691 @virajprabhu Large LSTM network for Text Generation using keras.
https://machinelearningmastery.com/text-generation-lstm-recurrent-neural-networks-python-keras/
https://github.com/Ram81/Deep_Learning_Models/blob/master/lstm_text_generation.py

keras

### CODE REVIEW

**Comment by virajprabhu** [comment]
@Ram81 Thanks! Can you please update the PR description with a stable link to the model for which you are adding support?

**Comment by Ram81** [comment]
@virajprabhu is this description enough?

**Comment by utsavgarg** [comment]
@Ram81 You are importing the model using Keras, but the export to Keras is not working. Please make sure that at least the export to the framework you are importing from works.

**Comment by Ram81** [comment]
@utsavgarg I will have a look into it

**Comment by Ram81** [comment]
@utsavgarg Done :+1:

File: keras_app/views/layers_export.py [hunk 1]
```diff
@@ -414,6 +414,7 @@ def recurrent(layer, layer_in, layerId):
     use_bias = layer['params']['use_bias']
     dropout = layer['params']['dropout']
     recurrent_dropout = layer['params']['recurrent_dropout']
+    return_sequences = True
```
Comment (utsavgarg) [inline keras_app/views/layers_export.py]: If you took in this value during import, why did you hardcode it for export? I am changing it for you.

Comment (Ram81) [reply keras_app/views/layers_export.py]: @utsavgarg Sorry I forgot to commit that changed file :sweat_smile:

**Pull request merged**